package.json

```diff
@@ -45,15 +67,16 @@
     "serverless": "^1.42.2 || ^2.0.0 || ^3.0.0"
   },
   "scripts": {
-    "build": "rollup -c",
+    "lint": "eslint --cache .",
```

Comment:
You can lint just the staged files during git commits using a setup like this: https://laurieontech.com/posts/husky/

No pressure to use it, it can cause issues with vscode git tools.

Reply:
Yeah I think that's fine for now!